snmp/tests/compose/data/cisco-nexus.snmprec

```diff
@@ -3250,7 +3250,7 @@
 1.3.6.1.4.1.9.9.500.1.2.1.1.5.21|66|55695
 1.3.6.1.4.1.9.9.500.1.2.1.1.6.21|2|8
 1.3.6.1.4.1.9.9.500.1.2.1.1.7.21|4x|ffffffffffff
-1.3.6.1.4.1.9.9.500.1.2.2.1.1.7|2|3
+1.3.6.1.4.1.9.9.500.1.2.2.1.1.2|2|3
```

Comment:
Note for reviewers: the index is the ifIndex, `2` ifIndex exist, but `7` does not.

Reply:
Not sure but this might need to be sorted to overcome an `snmpsim` limitation https://datadoghq.dev/integrations-core/tutorials/snmp/sim-format/#file-contents

> Due to a limitation of snmpsim, contents of .snmprec files must be sorted in lexicographic order.
Use $ sort -V /path/to/profile.snmprec to sort lines from the terminal.

Reply:
@pducolin I think it's already sorted and the change made doesn't change the order.

If it's not sorted, it should raise this error: https://github.com/DataDog/docker-library/blob/master/snmp/entrypoint.sh#L6-L9 (snmpsim docker image we use)

Reply:
True

1.3.6.1.4.1.9.9.500.1.2.**2**.1.1.2 > 1.3.6.1.4.1.9.9.500.1.2.**1**.1.7.21